versions/3.0.md

```diff
@@ -1050,6 +1051,36 @@ Response with no return value:
 description: object created
 ```
 
+#### <a name="callbackObject"></a>Callback Object
```

Comment:
This is called here »Callback Object«, but »Callback Responses Object« in the Operation object.